tslint.json

```diff
@@ -1,28 +1,67 @@
 {
-  "extends": ["tslint:latest", "tslint-react", "tslint-config-prettier", "tslint-plugin-prettier"],
+  "extends": [
+    "tslint:latest",
+    "tslint-react",
+    "tslint-config-prettier",
+    "tslint-plugin-prettier"
+  ],
+  "defaultSeverity": "error",
   "rules": {
-    "ban": [
-      true,
-      ["Object", "assign", "use TS2.1 object spread { ...a, ...b }"],
-      ["Object", "getOwnPropertyNames", "use Object.keys()"],
-      ["describe", "only"],
-      ["it", "only"]
-    ],
+    "ban": {
+      "options": [
+        ["Object", "assign", "use TS2.1 object spread { ...a, ...b }"],
+        ["Object", "getOwnPropertyNames", "use Object.keys()"],
+        ["describe", "only", "should not be committed to repo"],
+        ["it", "only", "should not be committed to repo"],
+        ["test", "only", "should not be committed to repo"]
+        // TODO: enable
+        // ["assert", "equal", "use assert.strictEqual instead"]
+      ]
+    },
     "file-header": {
-      "options": ["@license Copyright \\d{4} Palantir Technologies, Inc\\. All rights reserved."]
+      "options": ["Copyright \\d{4} Palantir Technologies, Inc\\. All rights reserved."]
+    },
+    "jsx-key": false,
+    "linebreak-style": {
+      "options": ["LF"]
+    },
+    "no-console": {
+      "options": ["log", "time", "timeEnd", "trace"]
+    },
+    "no-default-export": true,
+    "no-invalid-this": {
+      "options": ["check-function-in-method"]
+    },
+    "no-submodule-imports": {
+      "options": [
+        "documentalist",
+        "lodash",
+        "react-dom"
+      ]
+    },
+    "no-unnecessary-callback-wrapper": true,
+    "no-unnecessary-initializer": true,
+    // TODO: enable no-unused-expression. Currently, a lot of chai assertions are not writtten as proper statements.
```

Comment:
Example? Do you mean something like `expect(blah).to.be.true;`?

Reply:
yes